### PR TITLE
chore(zh-cn): replace google developers sites with CN mirror

### PR DESCRIPTION
**Pull request by yin1999**
### Description

replace google developers sites with CN mirror

### Additional details

- web.dev CN mirror: https://web.dev/blog/web-dev-for-china

### Related issues and pull requests

Part of: #24643

### CODE REVIEW

File: files/zh-cn/web/progressive_web_apps/index.md [hunk 1]
```diff
@@ -98,6 +98,6 @@ Service worker 可以使用以下 API 来使你的应用程序在离线状态下
 
 ## 参见
 
-- web.dev 上的[渐进式 web 应用](https://web.dev/explore/progressive-web-apps)
-- web.dev 上的[学习 PWA](https://web.dev/learn/pwa/)
+- web.dev 上的[渐进式 web 应用](https://web.developers.google.cn/explore/progressive-web-apps)
+- web.dev 上的[学习 PWA](https://web.developers.google.cn/learn/pwa/)
```
Comment (yin1999) [inline files/zh-cn/web/progressive_web_apps/index.md:102]:
```suggestion
- web.developers.google.cn 上的[渐进式 web 应用](https://web.developers.google.cn/explore/progressive-web-apps)
- web.developers.google.cn 上的[学习 PWA](https://web.developers.google.cn/learn/pwa/)
```

File: files/zh-cn/web/progressive_web_apps/tutorials/js13kgames/index.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ PWA 赋予了我们创建同时拥有以上两种优势的应用的能力。
 
 正如前文所述，PWA 不是只使用一种技术创建的。它代表了构建 Web 应用程序的新理念，涉及一些特定的模式，API 和其他功能。一眼是看不出来一个 Web App 是不是 PWA 的。如果应用程序满足某些要求，或者实现了一组特定的功能，例如离线工作、可安装、易于同步、可以发送推送通知等，我们就可以将其视为 PWA。
 
-此外，还有一些工具可以按百分比衡量应用的完整性。（[Lighthouse](https://developers.google.com/web/tools/lighthouse/)目前是最受欢迎的工具）通过实施各种技术优势，我们可以使应用程序更加渐进式，从而最终获得更高的 Lighthouse 得分。但这只是一个粗略的指标。
+此外，还有一些工具可以按百分比衡量应用的完整性。（[Lighthouse](https://developer.chrome.google.cn/docs/lighthouse/overview)目前是最受欢迎的工具）通过实施各种技术优势，我们可以使应用程序更加渐进式，从而最终获得更高的 Lighthouse 得分。但这只是一个粗略的指标。
```
Comment (yin1999) [inline files/zh-cn/web/progressive_web_apps/tutorials/js13kgames/index.md:28]:
```suggestion
此外，还有一些工具可以按百分比衡量应用的完整性。（[Lighthouse](https://developer.chrome.google.cn/docs/lighthouse/overview) 目前是最受欢迎的工具）通过实施各种技术优势，我们可以使应用程序更加渐进式，从而最终获得更高的 Lighthouse 得分。但这只是一个粗略的指标。
```

File: files/zh-cn/web/security/same-origin_policy/index.md [hunk 3]
```diff
@@ -143,6 +143,6 @@ JavaScript 的 API 中，如 {{domxref("HTMLIFrameElement.contentWindow", "ifram
 ## 参见
 
 - [W3C 介绍的同源策略](https://www.w3.org/Security/wiki/Same_Origin_Policy)
-- [web.dev 介绍的同源策略](https://web.dev/articles/same-origin-policy)
+- [web.dev 介绍的同源策略](https://web.developers.google.cn/articles/same-origin-policy)
```
Comment (yin1999) [inline files/zh-cn/web/security/same-origin_policy/index.md:146]:
```suggestion
- [web.developers.google.cn 介绍的同源策略](https://web.developers.google.cn/articles/same-origin-policy)
```

**Comment by fuchunhui** [comment]
佐证材料，中文镜像发布：https://mp.weixin.qq.com/s/OK-07avMAAytS2SVwrDVdg

File: files/zh-cn/web/html/global_attributes/itemscope/index.md [hunk 4]
```diff
@@ -241,7 +241,7 @@ Directions: <br>
 </table>
 
 > [!NOTE]
-> Google 提供了一个方便地从 HTML 提取微数据结构的工具：[Structured Data Testing Tool](https://developers.google.com/structured-data/testing-tool/)。你可以通过上面的 HTML 试一下。
+> Google 提供了一个方便地从 HTML 提取微数据结构的工具：[富文本搜索结果测试工具](https://search.google.com/test/rich-results)。你可以通过上面的 HTML 试一下。
```
Comment (jasonren0403) [inline files/zh-cn/web/html/global_attributes/itemscope/index.md:244]: 和其他 item* 文档统一下？

```suggestion
> Google 提供了一个方便地从 HTML 提取微数据结构的工具：[结构化数据测试工具](https://developers.google.cn/search/docs/appearance/structured-data)。你可以通过上面的 HTML 试一下。
```

**Review by jasonren0403 — COMMENTED** [review]
检查时发现的一些问题

File: files/zh-cn/web/api/navigator/scheduling/index.md [hunk 5]
```diff
@@ -29,4 +29,4 @@ l10n:
 
 - engineering.fb.com 上的 [Facebook 在 2019 年对浏览器 API 的首次贡献](https://engineering.fb.com/2019/04/22/developer-tools/isinputpending-api/)（2019）
 - developer.chrome.google.cn 上的[使用 isInputPending() 改善 JS 调度](https://developer.chrome.google.cn/docs/capabilities/web-apis/isinputpending)（2020）
-- web.dev 上的[优化耗时较长的任务](https://web.dev/articles/optimize-long-tasks#yield_only_when_necessary)（2022）
+- web.developers.google.cn 上的[优化耗时较长的任务](https://web.developers.google.cn/articles/optimize-long-tasks#yield_only_when_necessary)（2022）
```
[the remark this answers is not in the document]
Comment (yin1999) [reply files/zh-cn/web/api/navigator/scheduling/index.md:32]:
```suggestion
- web.developers.google.cn 上的[优化耗时较长的任务](https://web.developers.google.cn/articles/optimize-long-tasks)（2022）
```

File: files/zh-cn/web/html/global_attributes/itemscope/index.md (hunk 4 above)
Comment (yin1999) [reply files/zh-cn/web/html/global_attributes/itemscope/index.md]: 可以，就暂时不修改英文文档了（周期拉得会比较长）

**Comment by jasonren0403** [comment]
还有一个地方导致 CI 不过：

`files/zh-cn/web/api/htmllinkelement/fetchpriority/index.md` L53

**Comment by yin1999** [comment]
> 还有一个地方导致 CI 不过：
> 
> `files/zh-cn/web/api/htmllinkelement/fetchpriority/index.md` L53

Done

File: files/zh-cn/glossary/time_to_interactive/index.md [hunk 6]
```diff
@@ -19,4 +19,4 @@ TTI 是通过利用[长任务 API](/zh-CN/docs/Web/API/PerformanceLongTaskTiming
 
 - Web 孵化器社区组关于 [TTI 的定义](https://github.com/WICG/time-to-interactive)
 - Radimir Bitsov 写的[可交互时间——以人为本的单位](https://calibreapp.com/blog/time-to-interactive)
-- [计算 TTI](https://web.dev/articles/user-centric-performance-metrics#tracking_tti)
+- [计算 TTI](https://web.developers.google.cn/articles/user-centric-performance-metrics#tracking_tti)
```
[the remark this answers is not in the document]
Comment (PassionPenguin) [reply files/zh-cn/glossary/time_to_interactive/index.md]: Time to Interactive (TTI) has proved overly sensitive to outlier network requests and long tasks, resulting in high variability in this metric. TTI was [removed as a metric from Lighthouse 10](https://developer.chrome.com/blog/lighthouse-10-0/#scoring-changes). Newer, alternative, metrics like [Largest Contentful Paint (LCP)](https://web.dev/articles/lcp), [Total Blocking Time (TBT)](https://web.dev/articles/tbt), and [Interaction to Next Paint (INP)](https://web.dev/articles/inp) are usually better metrics to use in place of TTI.

**Comment by PassionPenguin** [comment]
https://github.com/mdn/translated-content/pull/24840#discussion_r1867135061

这个链接直接删掉吧，因为 Chrome 在 Lighthouse 10 把 TTI 从性能指标中移除了，同时移除了相应的文档？

**Review by yin1999 — COMMENTED** [review]
鉴于所有的审查意见已经解决，我先合并当前 PR 了

**Pull request merged**